generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
I guess` spring.cloud.config.uri` property can still be overridden by command line option.  Also I'm wondering whether the bootstrap'ed app can still be run in standalone (without a cloud config server), it could be a requirement for integration tests.